src/main.ts

```diff
@@ -0,0 +1,15 @@
+export function add(a: number, b: number): number {
+  return a + b;
+}
+
+export function subtract(a: number, b: number): number {
+  return a - b;
+}
+
+export function multiply(a: number, b: number): number {
+  return a * b;
+}
+
+export function divide(a: number, b: number): number {
+  return a / b;
+}
```

Comment:
추론될 수 있는 타입은 생략하는 게, 코드를 더 간결하게 쓸 수 있어서 좋지 않을까요?
```suggestion
export function add(a: number, b: number) {
  return a + b;
}

export function subtract(a: number, b: number) {
  return a - b;
}

export function multiply(a: number, b: number) {
  return a * b;
}

export function divide(a: number, b: number) {
  return a / b;
}
```

Reply:
확인했습니다 확인하고 다시 PR하겠습니다